src/lib/targeting.test.ts

```diff
@@ -0,0 +1,127 @@
+import { isEpicContent, isEpicSuitable, isEpicWorthwhile } from './targeting';
+
+describe('The isEpicContent function', () => {
```

Comment:
Totally personal preference, but I'd probably write `describe('isEpicContent', () => ...` here, just to reduce the number of words you have to parse when reading this.

Reply:
Fair point, I was trying to make it all ready like one sentence but could probably declutter for that reason.